docs/software/dns_servers.md

```diff
@@ -81,33 +76,22 @@ Pi-hole is a DNS sinkhole with web interface that will block ads for any device
 
     The DietPi Pi-hole implementation uses the official installer script, but it comes with a few differences, compared to the official default setup:
 
-    1. The `/var/log/pihole.log` plain text DNS query log is disabled by default. It is a second query log implementation, as `/etc/pihole/pihole-FTL.db` is used as a database-wise log implementation already, used by the web interface to search long-term logs. If you however want to use the `pihole -t`/`pihole tail` command, to print colourised logs to console, you need to re-enable the file-based logging:
-
-        ```sh
-        pihole -l on
-        ```
-
-        Also the DietPi [logging system](../dietpi_tools/software_installation.md#log-system) needs to be changed, to disable DietPi-RAMlog, as otherwise `/var/log/pihole.log` is cleared hourly.
-    2. The logging duration for the database-wise DNS query log in `/etc/pihole/pihole-FTL.db` is reduced from 365 days to 2 days. An internal discussion revealed that no-one of us uses logs old than a few hours. One year of logs leads to database sizes from hundreds of MiBs to GiBs. We leave it at 2 days so that web interface dashboard graphs/diagrams are not empty after Pi-hole (re)starts. You can easily adjust the logging duration by editing the `/etc/pihole/pihole-FTL.conf` config file. E.g. to restore the default 365 days of logs:
-
-        ```sh
-        MAXDBDAYS=365
-        ```
+    1. The Pi-hole web UI port is set to 8089, i.e. it can be reached via: `http://<your.IP>:8089/admin/`
+    1. The Pi-hole web UI is secured with the global software password you chose during first run setup, default: `dietpi`
+    1. DNS query logging to `/var/log/pihole/pihole.log` is disabled. This does not affect the query logs in the web UI and database, but the `pihole -t`/`pihole tail` command does not show DNS queries anymore. If you want to use this command or need query logs in `/var/log/pihole/pihole.log` for other reasons, it can be re-enabled via web UI privacy settings or running: `sudo pihole-FTL --config dns.queryLogging true`
+    1. DNS query logging to database (as shown in web UI) is reduced to 2 days. This can be changed via web UI privacy settings or e.g. `sudo pihole-FTL --config database.maxDBdays 7` to raise it to 7 days.
```

Comment:
```suggestion
    1. DNS query logging to database (as shown in web UI) is reduced to 2 days. This can be changed via web UI privacy settings or e.g. 
    
        ```sh
        sudo pihole-FTL --config database.maxDBdays 7
        ```
        
        to raise it to 7 days.
```